components/Header/index.jsx

```diff
@@ -1,66 +1,170 @@
-      <ul className="flex justify-around py-4 mb-5">
-        <li>
-          <Link href="/" className="text-lg font-semibold">
-            Home
-          </Link>
-        </li>
-        <li>
-          <Scroll
-            className="text-lg font-semibold"
-            to="eligibility"
-            spy={true}
-            smooth={true}
-            offset={10}
-            duration={500}
-            delay={100}
-          >
-            Eligibility
-          </Scroll>
-        </li>
-        <li>
-          <Scroll
-            className="text-lg font-semibold"
-            to="timeline"
-            spy={true}
-            smooth={true}
-            offset={10}
-            duration={500}
-            delay={100}
-          >
-            Timeline
-          </Scroll>
-        </li>
-        {/* Teams - no plans yet */}
-        <li>
-          <Scroll
-            className="text-lg font-semibold"
-            to="gallery"
-            spy={true}
-            smooth={true}
-            offset={10}
-            duration={500}
-            delay={100}
+      <div className="flex flex-col items-center flex-wrap">
+        <Link href="/">
+          <Image
+            src={logomain}
+            alt="logo-main"
+            width="309"
+            height="358"
+            className="cursor-pointer mt-4"
+            priority
+          />
+        </Link>
+        <div className="flex justify-between items-center h-full w-full px-4 md:px-[120px] lg:px-[202px] my-5">
+          {/* nav bar links */}
+          <div className="hidden lg:flex">
+            <ul className="flex justify-around gap-12 items-center">
+              {navItems.map((item, index) => (
+                <li key={index}>
+                  {item.link ? (
+                    <Link href={item.link} className="text-lg font-semibold">
+                      {item.name}
+                    </Link>
+                  ) : (
+                    <Scroll
+                      className="text-lg font-semibold"
+                      to={item.to}
+                      spy={true}
+                      smooth={true}
+                      offset={10}
+                      duration={500}
+                      delay={100}
+                    >
+                      {item.name}
+                    </Scroll>
+                  )}
+                </li>
+              ))}
+              {/* Contact - redirect to facebook page */}
+            </ul>
+          </div>
+          {/* menu icon */}
+          <div className="lg:hidden">
+            <Image
+              src={menu}
+              alt="menu"
+              className="cursor-pointer w-6 h-6 sm:w-8 sm:h-8"
+              onClick={handleNav}
+            />
+          </div>
+          {/* social media */}
+          <div className="flex items-center justify-center">
+            {socialMedia.map((social, index) => (
+              <Link href={social.link} key={index}>
+                <Image
```

Comment:
When hovering the text element, the cursor should be a pointer. Use tailwindcss inline styles to add that.